packit_service/worker/testing_farm.py

```diff
@@ -118,6 +118,17 @@ def run_testing_farm_on_all(self):
         )
 
     def run_testing_farm(self, chroot: str) -> TaskResults:
+        # inform users about temporarily disabled testing farm
+        self.report_status_to_test_for_chroot(
+            state=CommitStatus.success,
+            description="Testing farm is temporarily disabled.",
+            chroot=chroot,
+            url="https://github.com/packit/packit/issues/1078",
```

Comment:
What about using this one (https://github.com/packit/packit-service/issues/803) and update the top comment?

Reply:
My point is to have a very clear message about what is a problem and what needs to be done to make it work.
(And we can edit the message in the meantime.)

I am not sure if the conversation in the issue 1078 clearly describes the problem.

Reply:
yes, probably #803 is more clear, changing the link to it